specutils/fitting/continuum.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import division
+
+import itertools
+import operator
+
+from astropy.modeling.polynomial import Chebyshev1D
+from astropy.modeling.fitting import SLSQPLSQFitter
+
+from ..fitting import fit_lines
+from ..manipulation.smoothing import median_smooth
+
+
+__all__ = ['fit_continuum', 'fit_continuum_generic']
+
+
+def fit_continuum_generic(spectrum, model=Chebyshev1D(3), fitter=SLSQPLSQFitter(),
```

Comment:
What about `fit_generic_continuum` instead?  I get the motivation to have parallel names, but in general I think it's better to use "real words" - that is, make the code literally readable as english where practical.

Reply:
changed